routing/router.go

```diff
@@ -66,6 +67,13 @@ func NewRouter() *mux.Router {
 	return rootRouter
 }
 
+func disableBrowserCache(next http.Handler) http.Handler {
```

Comment:
But could this affect also to the UI resources ?
I mean, if this is only to not cache the API calls, it sounds correct.
But if this could have side effect to re-fetch a JS/CSS/image resource, perhaps could have a penalty in performance.

Just thinking loud, not tested but worth to take a look.

Reply:
I had the same thoughts as @lucasponce

Reply:
Yesterday I tested, and had the idea that the FileServer handler was resetting the headers, but it doesn't. Probably I got confused with the request headers in the browser's console.

I moved the code to the handlers. It should only affect the /api endpoints now.